### PR TITLE
Added tests for sponsor section of event page

### LINKED ISSUE #1646: Add Tests for sponsor section of event page.
Add Tests for sponsor section of event page.

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes #1646 
Parent Issue #1614

### CODE REVIEW

**Comment by agbilotia1998** [comment]
@geekyd please review!

**Comment by agbilotia1998** [comment]
@geekyd the travis build is failing because on link is broken on event page the link is `http://mbm.asia`.Please check.

File: test/serverTest.js [hunk 1]
```diff
@@ -599,6 +599,24 @@ describe("Running Selenium tests on Chrome Driver", function () {
...
+      eventPage.checkSponsorSection().then(function () {
+        done();
+      }).catch(function (err) {
+        done(err);
+      });
+    });
+    
+    it('Checking broken links in of Sponsors section', function (done) {
+      eventPage.getSponsorsBrokenLinks().then(function(brokenLinksCount){
+        assert.equal(brokenLinksCount,0);
```
Comment (geekyd) [inline test/serverTest.js]: There is one broken link.

**Comment by geekyd** [comment]
@agbilotia1998 Yes The site does not exist.

**Comment by agbilotia1998** [comment]
@geekyd made changes please review.

**Comment by agbilotia1998** [comment]
@geekyd please review.

**Comment by agbilotia1998** [comment]
@geekyd please run a travis build here again.

**Comment by agbilotia1998** [comment]
@geekyd please review.

**Pull request merged**